docs/how_to_guides/ed_Editor.md

```
@@ -0,0 +1,211 @@
+```python
+github_token="...."
+acp="..."
+
+config = {
+  "connectorProxyUrl": acp,
+  "sync": {
+    "github": {
+      "repos": [
+        {
+          "owner": "your_name_or_company",
+          "repo": "your_process_model_repo",
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)